### PR TITLE
Examples update

### LINKED ISSUE #142: Performance improvement might not be genuine due to warm up issues
In the examples, the first run could be slow due to cuda warmup issues.

### PR DESCRIPTION
**Pull request by XbzOnGit**
This resolves #142 

Update examples with prewarm if we are comparing the first with the second.  
Add some numbers in README.md  
Add vllm prefix cache example

### CODE REVIEW

File: lmcache/cache_engine.py [hunk 1]
```diff
@@ -301,17 +313,30 @@ def retrieve(
...
 
         Output:
             kv_tensors: the kv cache of the tokens, in the format of nested 
                         tuples. Will be an empty tuple if no kv cache is 
                         retrieved.
 
-            num_tokens: the number of tokens in the kv cache
+            ret_mask: indicate which tokens are retrieved
         """
+        chunked_skip_len = 0
```
Comment (ApostaC) [inline lmcache/cache_engine.py:327]: The changes here seem to conflict with #153. Did you pulled/picked some old codes from there?

**Review by ApostaC — COMMENTED** [review]
Thanks Sixian! One quick question about the changes in the `cache_engine.py`:

File: lmcache/cache_engine.py (hunk 1 above)
Comment (YaoJiayi) [reply lmcache/cache_engine.py:327]: Another side note. @KuntaiDu and I have talked about these examples. They contain too much repetitive code. I'll refactor them later.

**Review by YaoJiayi — COMMENTED** [review]
Another side note. @KuntaiDu and I have talked about these examples. They contain too much repetitive code. I'll refactor them later.

File: lmcache/cache_engine.py (hunk 1 above)
Comment (XbzOnGit) [reply lmcache/cache_engine.py:327]: There are new commits after I wrote this PR. I think simply keeping the code in #153 should work.

**Pull request closed without merging**